src/python/WMCore/Services/WMArchive/DataMap.py

```diff
@@ -113,7 +112,6 @@
         # 'StageOutCommand': ''
     }],
     'WMCMSSWSubprocess': {},
-    'WMTiming': {},
```

Comment:
Are you removing this because this isn't yet fully functional? If so, then don't you prefer to leave it in while we work on getting this WMTiming metric in the proper fwjr document?

Reply:
I am removing this because it does not belong to this step part of the report and it should be at top level. But so far I do not see any defaults for top level parts in this codebase, and only defaults for step parts.